tests/integration/apps/basic_app.py

```diff
@@ -1,20 +1,21 @@
 import asyncio
 import time
-from typing import List, Dict, Any
-from lcserve import serving
+from typing import Dict, List
 
 import aiofiles
-from fastapi import WebSocket, UploadFile
+from fastapi import UploadFile, WebSocket
+
+from lcserve import serving
 
 
 @serving
-def sync_http(interval: int) -> str:
+def sync_http(interval: int, **kwargs) -> str:
```

Comment:
Breaking change. Let's pass the tracing handler with `func_data` only if `**kwargs` is passed.

Reply:
Handled, I still need to check if more llm data can be provided in tracing and add tests